Sources/Sentry/include/SentryUIApplication.h

```diff
@@ -0,0 +1,27 @@
+#import "SentryDefines.h"
+#import <Foundation/Foundation.h>
+
+#if SENTRY_HAS_UIKIT
+#    import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * A helper tool to retrieve informations from the application instance.
+ */
+@interface SentryUIApplication : NSObject
+
+/**
+ * Application shared UIApplication instance.
+ */
+@property (class, nonatomic, readonly, nullable) UIApplication *sharedApplication;
```

Comment:
`l`: Do these have to be `class`? Can't we create an instance of `SentryUIApplication`, make it a singleton in the dependency container, and use it wherever we need it as a dependency? This would make testing easier.

Reply:
Not, it does not need to be a `class` property. I was just following `UIApplication` class implementation.